monolithe/generators/lang/java/templates/sdkinfo.java.tpl

```diff
@@ -0,0 +1,16 @@
+{{ header }}
+
+package {{ name }}.{{ version_string }};
+
+public interface SdkInfo {
+
+    double API_VERSION      = {{ version }};
+    String API_PREFIX       = "{{ api_prefix }}";
+    String PRODUCT_ACCRONYM = "{{ product_accronym }}";    
+    String PRODUCT_NAME     = "{{ product_name }}";    
+    String CLASS_PREFIX     = "{{ class_prefix }}";    
+    String NAME             = "{{ name }}";
+    String ROOT_API         = "{{root_api}}";
+    Class<?> ROOT_OBJECT_CLASS  = {{ class_prefix }}{{ root_api|capitalize }}.class;
```

Comment:
For the example program: SdkInfo.ROOT_OBJECT_CLASS  = Root.class, which is the right generated class name.
For VSD generated code: SdkInfo.ROOT_OBJECT_CLASS  = Me.class, which is also the right generated class name.

If there was an issue with generating the class name, the java compiler would report a compile error.